docs/eventing/broker/broker-event-delivery.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Event delivery"
+weight: 04
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/event-delivery/#configuring-broker-delivery
+  - /docs/eventing/event-delivery/#broker-support
```

Comment:
Not sure these are needed, maybe delete.
Can we have more than one aliases in different pages if a page is split into different places?